modules/mono/glue/GodotSharp/.editorconfig

```diff
@@ -1,8 +1,13 @@
 [**/Generated/**.cs]
+generated_code = true
```

Comment:
I think this may disable some analyzers that we may not want to disable.

Reply:
That's fair. To err on the side of caution, I've removed it